frontend/src/components/ActionBar.tsx

```
@@ -156,7 +156,7 @@ export default function ActionBar({
 
     return Object.entries(assets).reduce((acc, [resourceId, entry]) => {
       const filteredAssetsFromQuery = entry.assets.filter((asset: Model) =>
-        asset.name.toLowerCase().includes(searchQuery.toLowerCase())
+      asset.name && asset.name.toLowerCase().includes(searchQuery.toLowerCase())
```

why would this case happen? we should probably let this fail if we can't find a name to an asset in the tree

---

the main reason is if a chart is unnamed. That can happen. The Alternative would be a try to generate an arbitrary name on the ingestion side.